Use v7 sidekick

bohnertchris opened the pull request:
Sidekick configurator is deprecated. Added updated steps to configure v7 of the Sidekick. Also removed references to .json config files since current sample content contains none.

Graphic of setup flow will also need to be updated.

bohnertchris committed:
Use v7 sidekick

Sidekick configurator is deprecated. Added updated steps to configure v7 of the Sidekick. Also removed references to .json config files since current sample content contains none.

Graphic of setup flow will also need to be updated.

bdenham commented on src/content/docs/get-started/index.mdx:
```diff
@@ -366,28 +366,19 @@ Now we need to link your GitHub storefront repo (the code) to your storefront fo
-1. Select the **Add project** button to complete your Sidekick setup. You should see a message stating: "Your Sidekick is configured for this project."
-1. Return to your folder, select **ALL files** (including `.json` config files), select the **Sidekick** extension, and click the **Preview** button. Repeat this for all files in each folder. Select all the files again, but select the **Publish** button to publish these files to the CDN. These actions add your content to the internal (preview) and production (publish) Edge Delivery CDNs, where th ...
+1. Install Sidekick from the Chrome Web Store and pin the Sidekick extension to your browser's toolbar.
+1. Open your content folder and sign into the Sidekick.
```
```suggestion
1. Open your content folder. You should now see an overlay menu bar at the bottom of your content pages.
```

bdenham committed:
Merge branch 'develop' into patch-1

bdenham commented on src/content/docs/get-started/index.mdx:
```diff
@@ -366,28 +366,19 @@ Now we need to link your GitHub storefront repo (the code) to your storefront fo
-1. Return to your folder, select **ALL files** (including `.json` config files), select the **Sidekick** extension, and click the **Preview** button. Repeat this for all files in each folder. Select all the files again, but select the **Publish** button to publish these files to the CDN. These actions add your content to the internal (preview) and production (publish) Edge Delivery CDNs, where th ...
+1. Install Sidekick from the Chrome Web Store and pin the Sidekick extension to your browser's toolbar.
+1. Open your content folder and sign into the Sidekick.
+1. Click the hamburger menu of the Sidekick and then **Add this project**.
```
```suggestion
1. From the overlay menu, select **Sign-in**.
1. After you are signed in, select the **Options** icon (three horizontal lines ⫢) from the Sidekick menu and select **Add this project** from the options.
```

bdenham commented on src/content/docs/get-started/index.mdx:
```diff
@@ -366,28 +366,19 @@ Now we need to link your GitHub storefront repo (the code) to your storefront fo
+1. Install Sidekick from the Chrome Web Store and pin the Sidekick extension to your browser's toolbar.
+1. Open your content folder and sign into the Sidekick.
+1. Click the hamburger menu of the Sidekick and then **Add this project**.
+1. Return to your folder, select **ALL files**, select the **Sidekick** extension, and click the **Preview** button. Repeat this for all files in each folder and each subfolder. Select all the files again, but select the **Publish** button to publish these files to the CDN. These actions add your content to the internal (preview) and production (publish) Edge Delivery CDNs, where they can be down ...
```
```suggestion
1. In your content folder, select all content files, then select the **Preview** button from the Sidekick overlay at the bottom of your browser page.  Repeat this for all files in each folder and each subfolder. Select all the files again, but select the **Publish** button to publish these files to the CDN. These actions add your content to the internal (preview) and production (publish) Edge Delivery CDNs, where they can be downloaded and displayed in your browser for local development.
```

bdenham reviewed (changes requested):
@bohnertchris, see suggestions. I think we'll need a callout that mentions the new AEM Sidekick and adds an image or two of the new Sidekick overlay at the bottom of the page.

bohnertchris committed:
Update src/content/docs/get-started/index.mdx

Co-authored-by: Bruce Denham <brucedenham@mac.com>

bohnertchris committed:
Update src/content/docs/get-started/index.mdx

Co-authored-by: Bruce Denham <brucedenham@mac.com>

bohnertchris committed:
Update src/content/docs/get-started/index.mdx

Co-authored-by: Bruce Denham <brucedenham@mac.com>

bohnertchris committed:
Update index.mdx

Added link to Sidekick docs immediately, aligned terminology, broke out run-on steps.

bohnertchris commented:
@bdenham, I appreciate you reviewing me suggestions. When possible, I always eschew repeating content that is available elsewhere. Therefore I stuck just to the steps required since details how to install and use the sidekick is already well-documented. I defer to your judgement of your audience to decide if it is required to repeat details of the sidekick or add an image.

bdenham committed:
Merge branch 'develop' into patch-1
